### DISCUSSION TITLE
Info on ABCD twin data?

### DISCUSSION THREAD

**Discussion by landryhuffman** [now-i-know-my-abcd/docs#36] [Where in the World is…? (Help)]
Has anyone done or seen any twin studies with the ABCD data? Is there any more information on twins/triplets outside of the rel_relationship variable?

**Comment by sfreis** [comment]
Hi there, 

I've worked with the twin data in ABCD: 

https://onlinelibrary.wiley.com/doi/full/10.1111/desc.13150

And: 
https://www.frontiersin.org/articles/10.3389/fnhum.2022.863235/full

At the time, we calculated the zygosity classifications in-house with the genetic data, but it looks like ABCD has genetic zygosity in the acspsw03 file in the newest release and a file with information for calculating zygosity based on physical ratings in 
abcd_tztab01 


-- | --


<br class="Apple-interchange-newline">

**Comment by sfreis** [comment]
Hi there, 

Here is an update using ABCD's genetic zygosity variables: 

https://github.com/sfreis/ABCDfamilies/blob/main/README.md
